mod_cluster/extension/src/test/java/org/wildfly/extension/mod_cluster/ModClusterSubsystemTestCase.java

```diff
@@ -410,7 +427,7 @@ public void testSSL() throws Exception {
         Assert.assertEquals("SSL_DHE_DSS_WITH_3DES_EDE_CBC_SHA,SSL_RSA_WITH_RC4_128_MD5,SSL_RSA_WITH_RC4_128_SHA,SSL_RSA_WITH_3DES_EDE_CBC_SHA", ssl.get("cipher-suite").resolve().asString());
         Assert.assertEquals("mykeyalias", ssl.get("key-alias").resolve().asString());
         Assert.assertEquals("mypassword", ssl.get("password").resolve().asString());
-        Assert.assertEquals("TLS", ssl.get("protocol").resolve().asString());
```

Comment:
Validation-wise, this is never checked in the model so XSD should not enforce it, also the default is wrong, the PR for that: https://github.com/wildfly/wildfly/pull/7072

Regarding safety of protocols, this should be address in Elytron (e.g. right defaults) and brought to this subsystem once elytron is integrated and modcluster integrated with it https://issues.jboss.org/browse/WFLY-3884